multigeiger/display.cpp

```diff
@@ -28,25 +28,28 @@ void display_start_screen(void) {
   pu8x8->clear();
   if (isLoraBoard) {
     // Display is only 4 lines by 8 characters; lines counting from 2 to 5
-    pu8x8->setFont(u8x8_font_amstrad_cpc_extended_f);        // u8x8 can handle only 8x8, smaller font has no advantage
+    pu8x8->setFont(u8x8_font_victoriamedium8_r);
     for (int i = 2; i < 6; i++) {
       pu8x8->drawString(0, i, "        ");  // clear all 4 lines
     }
   }
 
   if (isLoraBoard) {
-    pu8x8->drawString(0, 2, "Geiger-");
-    pu8x8->drawString(0, 3, " Counter");
-    pu8x8->drawString(0, 4, "Version:");
+    pu8x8->setFont(u8x8_font_amstrad_cpc_extended_f);
+    pu8x8->drawString(0, 2, "Multi-");
+    pu8x8->drawString(0, 3, " Geiger");
+    pu8x8->setFont(u8x8_font_victoriamedium8_r);
     snprintf(line, 9, "%s", VERSION_STR);  // 8 chars + \0 termination
-    pu8x8->drawString(0, 5, line);
+    pu8x8->drawString(0, 4, line);
   } else {
     pu8x8->setFont(u8x8_font_amstrad_cpc_extended_f);
-    pu8x8->drawString(0, 0, "Geiger-Counter");
```

Comment:
I'ld prefer to just change this to `Multi-Geiger` or `MultiGeiger` and keep the underlining.

In any case, not `Multi-Geiger-Counter`.

Reply:
ok, will revert